### ISSUE TITLE
Mutations for related data

### ISSUE THREAD

**Issue by bora89** [tmeasday/create-graphql-server#47]
Hi, that is a great project, thank you for you time!

It seems like generated code respects only querying with  related data (likes, followers etc), but for mutations I do not see lets say `addLiked`, `removeFollower`. Is it not implemented yet or hard to program?
![](http://joxi.ru/82Q9wdt1VL842d?d=1)

**Comment by tmeasday** [comment]
Hi @bora89, yes I think it could be possible we would need to think about the best API. Are you interested in submitting a PR?

**Comment by bora89** [comment]
@tmeasday yes, I will investigate the code and see how that could be solved, if you have any ideas - write it here please.
